content/test/unit/service/validations/Validations.spec.ts

```diff
@@ -1,78 +1,65 @@
 import { ContentAuthenticator } from '@katalyst/content/service/auth/Authenticator'
 import { Deployment } from '@katalyst/content/service/deployments/DeploymentManager'
-import { ValidationContext } from '@katalyst/content/service/validations/ValidationContext'
+import { NoFailure } from '@katalyst/content/service/errors/FailedDeploymentsManager'
 import { Validations } from '@katalyst/content/service/validations/Validations'
+import {
+  DeploymentToValidate,
+  ExternalCalls,
+  ServerEnvironment,
+  ValidationArgs
+} from '@katalyst/content/service/validations/Validator'
 import { MockedAccessChecker } from '@katalyst/test-helpers/service/access/MockedAccessChecker'
-import { AuditInfo, Entity, EntityType, EntityVersion, Timestamp } from 'dcl-catalyst-commons'
-import { AuthChain, AuthLinkType } from 'dcl-crypto'
+import { AuditInfo, Entity, EntityType, EntityVersion, Pointer, Timestamp } from 'dcl-catalyst-commons'
 import * as EthCrypto from 'eth-crypto'
 import ms from 'ms'
 
 describe('Validations', function () {
-  it(`When a non uploaded hash is referenced, it is reported`, () => {
-    const entity = buildEntity({
-      content: new Map([
-        ['name-1', 'hash-1'],
-        ['name-2', 'hash-2']
-      ])
-    })
-    const validation = getValidatorWithMockedAccess()
-    validation.validateContent(entity, new Map([['hash-1', Buffer.from([])]]), new Map([]), ValidationContext.ALL)
+  const LEGACY_AUDIT_INFO = {
+    version: EntityVersion.V3,
+    deployedTimestamp: 10,
+    authChain: [],
+    migrationData: {
+      // This is used for migrations
+      originalVersion: EntityVersion.V2,
+      data: 'data'
+    }
+  }
 
-    expect(validation.getErrors().length).toBe(1)
-    expect(validation.getErrors()[0]).toBe(notAvailableHashMessage('hash-2'))
-  })
+  const LEGACY_ENTITY = buildEntity({ timestamp: 1000 })
```

Comment:
What is the difference between this entity and a new one? As both are created using `buildEntity`